controllers/ovsdbcluster_controller.go

```diff
@@ -192,7 +192,7 @@ func (r *OVSDBClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	//
 
 	if err := r.finalizeServers(ctx, cluster, servers, serverPods); err != nil {
-		if _, ok := err.(clusterKickError); ok {
+		if _, ok := errors.As(clusterKickError); ok {
```

Comment:
Needs rework, it should be something like:
```
		var ckerror clusterKickError
		if errors.As(err, &ckerror) {
			return ctrl.Result{RequeueAfter: time.Duration(10) * time.Second}, nil
		}
```

This will also need the import:
```
import (
        ...
	"errors"
        ...
)
```